pyproject.toml

```diff
@@ -10,9 +10,32 @@ homepage = "https://q-ctrl.com"
 repository = "https://github.com/qctrl/python-open-controls"
 documentation = "https://docs.q-ctrl.com/open-controls/references/qctrl-open-controls/"
 keywords = [
+    "black opal",
+    "boulder opal",
+    "nisq",
```

Comment:
Do we need to add "fire opal" above?

Reply:
Oh true! I must have copied the old list by mistake. Thanks for spotting the missing line